Orchestrator/docs/NLRModels.md

```diff
@@ -1,19 +1,69 @@
 # Prebuilt Language Models
 
-Prebuilt language models have been trained towards more sophisticated tasks for both monolingual as well as multilingual scenarios. In public preview only English models are made available.
+Prebuilt language models have been trained towards more sophisticated tasks for both monolingual as well as multilingual scenarios,
+including intent prediction and entity extraction.
```

Comment:
Same, entity extraction is not commited yet

Reply:
Maybe suggest that entity models are only experimental at the moment (less commiting than preview).

Reply:
Add a sentence: "Entity extraction is currently experimental and not yet for production use."